repos/omeka-i18n-module/Module.php

```diff
@@ -46,13 +55,17 @@ public static function isFeatureFlagEnabled(ContainerInterface $container)
         }
 
         $globalSettings = $container->get('Omeka\Settings');
-        $localizationSettings = $globalSettings->get('i18n', []);
+        $transifexEnabled = boolval($globalSettings->get('i18n_transifex-enabled', false));
```

Comment:
Maybe forgetting something about Omeka setting namespacing, but what's with `_` then `-`?

Reply:
It's a mistake on my part really, when I created:
https://github.com/digirati-co-uk/madoc-platform/blob/master/repos/shared-library/src/Framework/AbstractConfigurationForm.php#L46

I modelled it off of a few examples of configuration that already existed, which used `NAMESPACE_KEY_NAME` as the pattern, so I when automating the saving side, just followed that pattern. It might make more sense to be `NAMESPACE/KEY-NAME` as the convention though. It would require a SQL migration for existing sites with the configuration though.